src/Symfony/Component/Debug/Tests/DebugClassLoaderTest.php

```diff
@@ -133,7 +133,8 @@ class_exists(__NAMESPACE__.'\TestingCaseMismatch', true);
      */
     public function testFileCaseMismatch()
     {
-        if (!file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
+        //Cannot be tested if the file loads (there is is casemismatch.php in the filesystem)
+        if (file_exists(__DIR__.'/Fixtures/CaseMismatch.php')) {
```

Comment:
Obviously it cannot be merged like this. The tests now fail on travis. So I guess the actual thing that this wants to test is broken and just changing the test will not solve the problem.